docs/core/tools/dotnet-install-script.md

```diff
@@ -15,7 +15,7 @@ ms.date: 09/11/2017
 
 Windows:
 
-`dotnet-install.ps1 [-Channel] [-Version] [-InstallDir] [-Architecture] [-SharedRuntime] [-DryRun] [-NoPath] [-AzureFeed] [-ProxyAddress] [--Verbose] [--Help]`
+`dotnet-install.ps1 [-Channel] [-Version] [-InstallDir] [-Architecture] [-SharedRuntime] [-DryRun] [-NoPath] [-AzureFeed] [-ProxyAddress [-ProxyUseDefaultCredentials]] [--Verbose] [--Help]`
```

Comment:
The `-ProxyUseDefaultCredentials` parameter is declared inside of another, it should outside of that: `[-ProxyAddress] [-ProxyUseDefaultCredentials]`

Reply:
My understanding is that `-ProxyUseDefaultCredentials` is only applicable if `-ProxyAddress` is also specified. 

Is that inaccurate? Or is there a better way to represent that restriction?

Reply:
Hrm yes I see what you mean! 

---- researching -----

I looked at other powershell docs and for example, https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-6 does not use that convention for parameters that depend on other parameters.